packages/vue/src/components/SpAssets/SpAssets.vue

```diff
@@ -57,26 +76,74 @@ export default defineComponent({
       )
     })
 
+    const filteredBalanceList = computed(() => {
```

Comment:
Since we also have a search going on [here](https://github.com/tendermint/vue/blob/e5bf4b3b7a241835253a13fa26aafbd8ba9451cc/packages/vue/src/components/SpAmountSelect/SpAmountSelect.vue#L182), we might as well come up with a composable, e.g. useSearch... 

Doesn't need to be in this PR though.

Reply:
Good idea. I also thought about a composable for search and we could send parameters like search criteria and more. Let's create a separate task for this later.